src/SideAppbar/SideAppbar.js

```diff
@@ -0,0 +1,130 @@
+import { Box, Link, Paper, Typography } from "@mui/material";
+
+import AppLogo from "./AppLogo";
+import PropTypes from "prop-types";
+import React from "react";
+
+// SideAppbar component for app which displays logo, list of items and app version
+function SideAppbar({ logoLinkUrl = null, showLogo = true, onButtonClick }) {
+  // use theme
+
+  // VIRTO app list
+  const appList = [
```

Comment:
Do we really want to hardcode the app list here? Thinking long term, not all customers might have access to all apps. We should pass in the array of apps during integration I think?

Thoughts @dmbarry86?

We can then also benefit from passing in the href so that we can use <Link /> properly rather than doing onButtonClick={(app) => navigate(app)}

Reply:
Passing the URL as a prop now.